patches/protonvpn-nm-lib/make-subprocess-wrapper-flatpak-aware.patch

```diff
@@ -0,0 +1,105 @@
+diff --git a/protonvpn_nm_lib/core/subprocess_wrapper.py b/protonvpn_nm_lib/core/subprocess_wrapper.py
+index 96634f6..9b2ed9e 100644
+--- a/protonvpn_nm_lib/core/subprocess_wrapper.py
++++ b/protonvpn_nm_lib/core/subprocess_wrapper.py
+@@ -1,5 +1,6 @@
+ import os
+ import platform
++import shutil
+ import subprocess as _subprocess
+ 
+ 
+@@ -25,9 +26,41 @@ class SubprocessWrapper:
+ 
+     @staticmethod
+     def is_root_owned(path):
++        if SubprocessWrapper.is_flatpak():
++            try:
++                owner_uid = int(_subprocess.check_output(
++                    ["flatpak-spawn", "--host", "stat", "-c", "%u", path]).decode().strip())
++                owner_gid = int(_subprocess.check_output(
++                    ["flatpak-spawn", "--host", "stat", "-c", "%g", path]).decode().strip())
++                return owner_uid == owner_gid == 0
++            except _subprocess.CalledProcessError:
++                pass
```

Comment:
Adding a giant sandbox escape to check if you are root makes no sense at all.

Anyway you should just assume that you are never root in flatpak. Flatpak is not meant to run as root and will not under normal usage.